detox/test/scripts/postinstall.js

```diff
@@ -47,6 +47,21 @@ function overrideReactAndroidGradleForRn64Android() {
   fs.copySync(PATCH_SCRIPT_PATH, REACT_ANDROID_GRADLE_SCRIPT_PATH);
 }
 
+function overrideReactAndroidGradleForRn66Android() {
```

Comment:
@jonathanmos where is `'ReactAndroid_rn66_build.gradle`, though?

Reply:
[it's in the same path](https://github.com/wix/Detox/tree/android-12/detox/test/scripts)

Reply:
Could you please add the `// Edited by Detox' post-install script` note to it?